plugins/modules/ec2_snapshot.py

```diff
@@ -70,6 +70,38 @@
     required: false
     default: 0
     type: int
+  modify_create_vol_permission:
+    description:
+      - If set to C(true), ec2 snapshot's createVolumePermissions can be modified.
+    required: false
+    type: bool
+    version_added: 6.0.0
+  purge_create_vol_permission:
+    description:
+      - Whether unspecified group names or user IDs should be removed from the snapshot createVolumePermission.
+      - Must set I(modify_create_vol_permission) to C(True) for when I(purge_create_vol_permission) is set to I(purge_create_vol_permission).
+    required: False
+    type: bool
+    default: False
+    version_added: 6.0.0
+  group_names:
+    description:
+      - The group to be added or removed. The possible value is C(all).
+      - Mutually exclusive with I(user_ids).
+    required: false
+    type: list
+    elements: str
+    version_added: 6.0.0
+  user_ids:
+    description:
+      - The account user IDs to be added or removed. The possible value is C(all).
```

Comment:
```suggestion
      - The account user IDs to be added or removed.
```
I'm assuming this was just a copy/paste error?

Reply:
fixed.